src/style-stars--5.css

```diff
@@ -0,0 +1,3 @@
+.stars--5 .stars__star {
+  background-image: url(images/star-active.svg);
```

Comment:
Remember to use fallback fonts as per the checklist. While this file doesn't deal with fonts directly, it's a good practice to ensure consistency across all style files in the project. Consider defining a fallback for the entire project in a base stylesheet.